src/sentry/integrations/vsts/integration.py

```diff
@@ -37,7 +37,8 @@ def get_client(self):
 
         return VstsApiClient(self.default_identity, VstsIntegrationProvider.oauth_redirect_url)
 
-    def get_project_config(self):
+    def get_default_project_config_field(self):
+        # Not going to be used at this time. Switching to last used project
```

Comment:
Then why not delete it?

Reply:
I can. That works :) I didn't know if we wanted to add it in at some later date. No Problem.

Reply:
We can always find it back in the commit history